db-protocol/postgresql/src/main/java/org/apache/shardingsphere/db/protocol/postgresql/packet/command/query/PostgreSQLRowDescriptionPacket.java

```diff
@@ -29,6 +30,7 @@
  * Row description packet for PostgreSQL.
  */
 @RequiredArgsConstructor
+@Getter
```

Comment:
Do we need this getter in production code? If not, please access this field by Mockito Plugins or reflection.

Reply:
only in test,  i'l remove it